✨(docker-compose) create a docker-compose config for example

lunika opened the pull request:
To ease the use of our images, we have created a docker-compose configuration.

lunika committed:
✨(docker-compose) create a docker-compose config for example

To ease the use of our images, we have created a docker-compose
configuration.

jmaupetit commented on example/docker-compose.yml:
```diff
@@ -0,0 +1,74 @@
+      - worker
+
+  api:
+    image: fundocker/learninglocker:v2.6.2
```
Why not using the `local` tag to use local builds?

lunika replied:
Because I want to use a published image

jmaupetit replied:
IMO the docker-compose configuration from this repository should stand for the project development purpose only. When developing another project that requires learning locker, you may use official/published images in your services, but not here.

jmaupetit commented on example/env.d/learninglocker:
```diff
@@ -0,0 +1,26 @@
+# Misc
```
Why about naming this file `env.d/learninglocker.example`? And adding a make rule to enable (copy) it?

lunika replied:
This docker-compose is here as an example, not to be used in this repo. This is why I created a real docker-compose project. If you want to use it you will copy it in your own project

jmaupetit replied:
But we need a docker-compose environment in this repository to test our build/modifications, no?

jmaupetit replied:
I think we need both.

lunika replied:
No I don't use it for tests, just for having an example how to use the image and how to build a complete learning locker stack.

jmaupetit replied:
Ok then. So it needs to be clearly stated in the documentation.

lunika replied:
It is in an `example` directory, I though it should be enough to understand that it is an... example ?

jmaupetit replied:
:joy: please, document this part. This is not that obvious for everyone that it should be copied in their own project.

lunika replied:
Done

jmaupetit commented on example/env.d/nginx:
```diff
@@ -0,0 +1,3 @@
+UI_NODE=ui:3000
```
Same remark about the file name here.

lunika committed:
📝(Readme) add a section explaining docker-compose config

The docker-compose config is here as an example in the repository. We
added a new section in the Readme explaining how to use this example.
